quarkus.rest-config instead of quarkus.rest-client

### Describe the bug

When using REST Client, when you forget to configure the uri of an external microservice, Quarkus displays an error message as follow:

```
Caused by: java.lang.IllegalArgumentException: Unable to determine the proper baseUrl/baseUri. 
Consider registering using @RegisterRestClient(baseUri="someuri"), @RegisterRestClient(configKey="orkey"), or by adding

'quarkus.rest-config."org.agoncal.fascicle.quarkus.book.client.NumberProxy".url' or 
'quarkus.rest-config."org.agoncal.fascicle.quarkus.book.client.NumberProxy".uri' to your Quarkus configuration
```

There is no such `quarkus.rest-config` property, but instead, `quarkus.rest-client`

### Expected behavior

The message should display the right properties

```
...
'quarkus.rest-client."org.agoncal.fascicle.quarkus.book.client.NumberProxy".url' or 
'quarkus.rest-client."org.agoncal.fascicle.quarkus.book.client.NumberProxy".uri' to your Quarkus configuration
```


### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

2.5.0

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/extensions/resteasy-classic/rest-client/runtime/src/main/java/io/quarkus/restclient/runtime/RestClientBase.java
+++ b/extensions/resteasy-classic/rest-client/runtime/src/main/java/io/quarkus/restclient/runtime/RestClientBase.java
@@ -29,8 +29,8 @@ import io.quarkus.restclient.config.RestClientsConfig;
 
 public class RestClientBase {
 
-    public static final String QUARKUS_CONFIG_REST_URL_FORMAT = "quarkus.rest-config.\"%s\".url";
-    public static final String QUARKUS_CONFIG_REST_URI_FORMAT = "quarkus.rest-config.\"%s\".uri";
+    public static final String QUARKUS_CONFIG_REST_URL_FORMAT = "quarkus.rest-client.\"%s\".url";
+    public static final String QUARKUS_CONFIG_REST_URI_FORMAT = "quarkus.rest-client.\"%s\".uri";
 
     private final Class<?> proxyType;
     private final String baseUriFromAnnotation;

